### ISSUE TITLE
Call tools

### ISSUE THREAD

**Issue by NickShargan** [orangeduck/Python-xNormal#1]
Is it possible to call tools of xNormal by this wrapper?

**Comment by orangeduck** [comment]
I don't believe so. Basically all this wrapper can do is just set the settings for a call to "Generate Maps"

**Comment by moppius** [comment]
You can run some tools (but not all of them, it seems). Running `xNormal.exe -h` shows a Help popup with some examples of other command lines.

For instance, you can pass these flags to run other tools:
* `-aogpu` - Simple AO GPU tool
* `-osbm` - OBJ to SMB converter
* `-os2ts` - Object space to tangent space converter

Support could be added for these tools specifically, but there's no documentation for the other tools.
The Object Space to Tangent Space converter seems the most useful one to me, was there any specific tool you wanted to call, @NickShargan?
